content/support/enablement/k8-migration.md

```diff
@@ -0,0 +1,101 @@
+
+- Generate the database dumps.
+
+  - Dumps for the primary **sourcegraph** DB
+    `kubectl exec -it $pgsql_POD_NAME -- bash -c 'pg_dump -C --username sg sg' > sourcegraph_db.out`
+
+  - Dumps for the **codeintel** DB
+    `kubectl exec -it $codeintel-db_POD_NAME -- bash -c 'pg_dump -C --username sg sg' > codeintel_db.out`
+
+- Ensure you copy/move the `sourcegraph_db.out` and `codeintel_db.out` files to the root of the `deploy-sourcegraph` directory of the new Sourcegraph deployment.
+
+### Steps to restore Sourcegraph Databases into a new environment
+
+In this section of the tutorial, you would learn how to restore the primary `sourcegraph` database and the `codeintel` database from the generated dumps in the previous section.
+
+- Before restoring on the new deployment, It is very important you stop all connections to the Database from the frontend.
+
+  -     Scale down all the pods in the deployment
+    `kubectl scale deployment ---all --reliplicas=0 `
+  -     Scale down all the statefulset
+    `kubectl scale sts ---all --reliplicas=0 `
+
+- Then restart ONLY the `pgsql` and `codeintel` pods
+
+  -     Start the pgsql:
+    `kubectl scale pgsql --reliplicas=1 `
+  -     Start the codeintel-db:
+    `kubectl scale codeintel-db --reliplicas=1 `
+
+- Copy the database files into the pods by running the following command from the root of the deploy-sourcegraph directory
+
+  ` kubectl cp sourcegraph_db.out $pgsql_POD_NAME:/tmp/sourcegraph_db.out`
+
+  `kubectl cp codeintel_db.out $codeintel-db_POD_NAME:/tmp/codeintel_db.out`
+
+- Exec into the pgsql pod and then the template1 database:
+
+  ```
+  kubectl exec -it <pgsql-pod-name> -- sh
+  $ psql -U sg template1
+  ```
+
+- Drop the sg database, recreate it, then exit the psql session (but still in the pod exec):
+
+  ```
+  # DROP DATABASE sg;
+  # CREATE DATABASE sg;
+  # \q
+  ```
+
+- Restore the database dump:
+
+  ```
+  $ psql -U sg -f /tmp/sourcegraph_db.out sg
+  $ psql -U sg -f /tmp/codeintel_db.out sg
+
+  ```
+
+- Exec into the pgsql and run ` SELECT * FROM REPO LIMIT 5;` to be sure the restoration was complete.
+
+- Run `kubectl apply --recursive -f base/` to apply the changes.
+- Run `kubectl get pods -o wide` to make sure your pods are running again.
+- Start the remaining Sourcegraph services by following the steps in [applying manifests](https://docs.sourcegraph.com/admin/install/kubernetes/operations#applying-manifests).
+
+### Additional Information
+
+- Since your new deployment was running on a port before the restore, there is a probability that you might lose connection. Run `sudo lsof -i -P | grep LISTEN` to see the services using the port and kill the `PID` with `kill -9 <PID_number>`.
+
+- You can now port forward your k8s deployment to localhost or wherever your deployment is hosted. For example, `kubectl port-forward svc/sourcegraph-frontend 3080:30080` (note you may need to add `-n ns-sourcegraph` to the command).
```

Comment:
Here as well.. maybe instead of `-n ns-sourcegraph` do `-n $NAMESPACE` so it makes sense in the context of the top comment?